# Test file with error
def bad(): x.append(1)
